docs/community/faq/embeddings.md

```diff
@@ -0,0 +1,15 @@
+# Embeddings
+
```

Comment:
Could we kindly make the following changes to the overall format.

- At the beginning of every page, let's have a sort of Table of Contents, but perhaps call it ToC
- We can drop `Q` and `A1:` identifiers. I think the section lines do a good job of segmenting these so we don't really need to be explicit with `Ax`.

As an example, see below:
```
# Embeddings

##### FAQ
1. [How to use a custom/local embedding model?](#1-how-to-use-a-customlocal-embedding-model)
2. [How to use a local hugging face embedding model?](#2-how-to-use-a-local-hugging-face-embedding-model)

---

##### 1. How to use a custom/local embedding model?

To create your customized embedding class you can follow [Custom Embeddings](../../examples/embeddings/custom_embeddings.ipynb) guide

---

##### 2. How to use a local hugging face embedding model?

To use a local HuggingFace embedding model you can follow [Local Embeddings with HuggingFace](../../examples/embeddings/huggingface.ipynb) guide

---

```

Reply:
![image](https://github.com/run-llama/llama_index/assets/92402603/0f4b3483-de39-4b2b-8fea-60e03e176f4d)

which should render as above ☝️